virtual/README.md

```diff
@@ -102,8 +97,7 @@ $ lspci -nnk | grep NVIDIA
 
```

Comment:
Also, add a line explaining source k8s_environment.sh

Reply:
`source k8s_environment.sh` enables us to run `kubectl` from the host without having to ssh into the vm, correct? where should this be added?
after [Start the Cluster](https://github.com/satindern/deepops/blob/patch-1/virtual/README.md#start-the-cluster)?

Reply:
It enables us to run kubectl on the provisioning node (where we ran the deepops scripts). When `./cluster_up.sh` runs, it copies a version of `kubectl` and `admin.conf` to the `k8s-config/artifacts` directory. The `k8s_environment.sh` script adds `kubectl` to the path and sets the `KUBECONFIG` variable to reference `admin.conf`. Afterwards, you can run `kubectl` commands against the k8s cluster.